src/Phinx/Migration/Manager.php

```diff
@@ -113,6 +113,7 @@ public function printStatus($environment, $format = null)
         $migrationCount = 0;
         $missingCount = 0;
         $pendingMigrationCount = 0;
+        $migrations_final = [];
```

Comment:
$finalMigrations maybe?

Reply:
that's better, and in the right case. Clearly did not give myself enough time to context switch between projects.